applications/ArtZero_InkWhale.md

```diff
@@ -0,0 +1,141 @@
+network. He is the founder of ArtZero - the first NFT Marketplace & also 1st dApp on Aleph Zero network.
+
+Frankie Kao owns a design company and have been working in many projects on web design, graphic designs etc. He dedicated his resources to work with ArtZero.
+
+Phuong Hoang has been in sale and marketing industry for over a decade. She has been Sale and Marketing Manager/Director for many companies including: Honda Vietnam, Plex Cinama, Saga Media etc.
+
+All of our developers have at least 5-10 years experience in software development.
+
+### Team Code Repos
+
+- https://github.com/artzero-io (some repos are in private mode)
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/nad128668/
+- http://linkedin.com/in/phuong-phoebe-hoang-4b5888148
+- https://www.linkedin.com/in/tran-albert-469b6319a
+
+## Development Status :open_book:
+
+We have completed the Testnet demo version of the ArtZero NFT Marketplace. The contracts are waiting to be audited then our platform will be ready to be launch on Aleph Zero Mainnet.
+
+https://artzero.io
+https://docs.artzero.io/
+
+Ink Whale is still in development and the MVP is completed for public test at https://inkwhale.net
+
+## Development Roadmap :nut_and_bolt:
+
+
+### Overview
+
+- **Total Estimated Duration:** 4 months
+- **Full-Time Equivalent (FTE):**  5 FTE
+- **Total Costs:** 30,000 USD
+
+
+### Milestone 1  — NFT Marketplace on Moonbeam and Astar
+* **Estimated Duration:** 8 weeks
+* **Costs:** 15,000 USD
+
+In this milestone, We will implement all Ink! Smart contracts on Moonbeam and Astar Testnet with full working NFT Marketplace UI/UX and backend. Consider to launch on Mainnet when the networks are ready with Mainnet Smart Contract.
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0a.    | License            | Apache License 2.0                                           |
+| 0b.    | Documentation      | We will provide technical documents and user guides |
+| 0c.    | Testing Guide      | We will provide Test Plan and Test Results for operating and using the NFT Marketplace |
+| 0d.    | Article/Tutorial   | We will write an article or tutorial that explains the NFT Marketplace |
+| 1. | smart contract implementation | Audited contracts to be deployed on Moonbeam and Astar. We use ink! to develop the contracts and the functions that contracts support are: list, unlist NFT, buy NFT at fixed price, bid for an NFT, accept an NFT bid, add NFT Collection, add a launchpad project. |
+| 2. | Backend | Build infrastructure to serve the need of NFT Marketplaces. We use nodejs and mongodb on AWS Services. |
+| 3. | Frontend | Clone front-end from ArtZero to work with Moonbeam and Astar|
```

Comment:
Are you adding functionality to the front-end or just cloning it? If you don't plan to make changes to it for the first milestone, can you remove this part, as we normally don't pay for simply cloning a repo.